### DISCUSSION TITLE
How can I uninstall/remove Parabolic on Windows 11?

### DISCUSSION THREAD

**Discussion by Verhoeckx** [NickvisionApps/Parabolic#930] [Q&A] [answered]
Hello Everybody,

How can I uninstall/remove Parabolic on Windows 11?

**Comment by nlogozzo** [comment]
![image](https://github.com/user-attachments/assets/eabb58d6-6413-4925-b97c-78cdf498a999)

Same as any other app...

**Comment by Verhoeckx** [comment]
I (already) searched for 'Parabolic' and 'Nickvision' in the section Installed apps in the Setting app, but it's not there. That's why I asked how I can manually remove it.

I found the app in the local hidden directory AppData and removed all the directories that I could find. Strange, but even then, it's still not completely removed (I can still start it). Which other directories does Parabolic use?

**Reply by nlogozzo** [reply]
It seems as if you didn't install the app correctly? Did you use the provided installer from the GitHub releases page?

**Reply by nlogozzo** [reply]
Do you remember which version of Parabolic you were using?

**Comment by Verhoeckx** [comment, marked as answer]
Hello @nlogozzo,

I followed the steps on Github to install the application:

1. Download and extract the NickvisionParabolic_V2024.10.3_Windows.zip file from the assets below
2. Run org.nickvision.tubeconverter.qt.exe

Oh, wait, I see there is also an executable file on Github! I followed the steps of the portable version. Sorry! Read to quick :-/.

**Reply by nlogozzo** [reply]
Yes, so you downloaded the portable version, and the browser probably extracted it to that AppData location and that's where it is being run from. Just deleting that folder will clear Parabolic the portable version.

**Reply by Verhoeckx** [reply]
I have done that, so now it's deleted! Thanks for your help!
